# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../ooo/package-ooo.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./bin/package-ooo.orig	2006-11-27 14:21:32.000000000 -0300
+++ ./bin/package-ooo	2006-11-27 14:21:13.000000000 -0300
@@ -460,11 +460,6 @@
     fi
 }
 
-if ! test -f $OODESTDIR/gid_Module_Root; then
-    echo "Failed to generate package file lists";
-    exit 1;
-fi
-
 cd $OODESTDIR
 
 if test -f gid_Module_Root_Files_2; then
@@ -742,9 +737,6 @@
 	find . -path './pkg' -prune -o -not -name 'gid_Module_*' -not -type d -exec echo "File not packaged: {}" \;
 fi
 
-echo "Cleaning up lists of files...";
-mv -f $OODESTDIR/gid_Module_* $BUILDDIR
-
 cd $BUILDDIR
 
 # mark the config files
